srcpkgs/libpipewire0.2/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'libpipewire0.2'
+pkgname=libpipewire0.2
+version=0.2.7
+revision=1
+wrksrc=pipewire-${version}
+build_style=meson
+configure_args="-Dgstreamer=disabled -Ddocs=false -Dsystemd=false"
+hostmakedepends="pkg-config"
+makedepends="dbus-devel alsa-lib-devel libudev-devel"
```

Comment:
libudev-devel isn't a package

Reply:
How did it build then? Or maybe the package name has changed recently

Reply:
Looks like `eudev-libudev-devel` provides `libudev-devel`... I'll switch it to that.